samples/django/django_user/polls/views.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.http import HttpResponse
+
+from oauth2client.contrib.django_util import decorators
+
+
+def index(request):
+    return HttpResponse("Hello world!")
+
+
+@decorators.oauth_required
+def get_profile_required(request):
```

Comment:
Sorry ignore, realised it's related to https://github.com/google/oauth2client/pull/636/files#r77437699